### PR TITLE
Sensor relay refit

### PR DESCRIPTION
**Pull request by nauticall**
Overhauls the sensor relay away site to be up to modern standards. It's now more practical, more decrepit, and has more facilities for things while retaining the old loot and goodies it had before. 

The place starts depowered, save for the airlocks, which are powered by default to allow entry without having to bust down a window/wall/inflatable and vent the place.

In addition this PR adds a special sprite for the sensor array the relay uses, and tweaks its stats:
* Now has a maximum range of 24.
* Has a stable range of 10.
* Cannot deep scan at all.
* Overheats very quickly at high ranges.
* Detects passing ships slower than a standard relay.

The above should make it more worthwhile to capture and use, as when datalinked it provides visibility of a large portion of the sector.

image

image
image

# Before powering
image

# After powering
<details><summary>Click to expand</summary>
<p>

image
image
image
image
image
image
image

</p>
</details>

### CODE REVIEW

**Comment by nauticall** [comment]
!review

**Comment by nauticall** [comment]
todo: remove old relay dmi and populate pr info

**Comment by DreamySkrell** [comment]
is this ready for review

title says it's WIP, it still needs changelog, there is a "todo: "comment on the pr

**Comment by nauticall** [comment]
> is this ready for review

it's ready now

**Review by Alberyk — APPROVED** [review]

File: code/modules/overmap/ships/computers/sensors.dm [hunk 1]
```diff
@@ -640,3 +640,16 @@
 		/obj/item/bluespace_crystal,
 		/obj/item/stack/cable_coil = 30
 	)
+
+/obj/machinery/shipsensors/strong/venator/relay
```
Comment (DreamySkrell) [inline code/modules/overmap/ships/computers/sensors.dm:644]: should be `/obj/machinery/shipsensors/strong/relay`
venator sensors are specific to horizon
and this subtype overrides all the vars anyways

Comment (DreamySkrell) [reply code/modules/overmap/ships/computers/sensors.dm:644]: should also have `component_types` and its own circuit board

Comment (DreamySkrell) [reply code/modules/overmap/ships/computers/sensors.dm:644]: it should also not scan further out than Horizon's sensors
horizon's sensors are supposed to be the best of the best, most high tech, etc
meanwhile some tiny old sensors on a tiny old station are better?

**Comment by DreamySkrell** [comment]
image
this can spawn in any sector, NKA comics are not fitting on it

File: code/modules/overmap/ships/computers/sensors.dm (hunk 1 above)
Comment (Eyeveri) [reply code/modules/overmap/ships/computers/sensors.dm:644]: I agree with this, the Horizon is supposed to have the most power sensors around. It was one of the selling points for the initial lore arc when it launched.

**Review by DreamySkrell — APPROVED** [review]

**Pull request merged**